README.md

```diff
@@ -225,6 +227,80 @@ When using backreferences, there is a couple of things to keep in mind:
 - `__via__` is surrounded with two underscores; you can read this query out loud like this: _"get posts that link to author via the author field"_
 
 
+## Helpers
+
+`cf-graphql` comes with included helpers. These are used inside of [the demo application](https://github.com/contentful-labs/cf-graphql/tree/master/demo).
```

Comment:
Agree with the change, still, think it's nice to say that they're used there for people to check out.